presidio-anonymizer/app.py

```diff
@@ -57,8 +57,10 @@ def anonymize():
             if not content:
                 return ErrorResponse("Invalid request json").to_json(), 400
 
-            data = AnonymizerRequest(content, self.engine.builtin_anonymizers)
-            text = self.engine.anonymize(data)
+            anonymizers = AnonymizerRequest.handle_anonymizers_json(content)
```

Comment:
nit: rename `anonymizers` to `anonymizers_config` to conform with what the engine is requesting. The whole config vs non config is somewhat confusing.

Reply:
Done.